e2e/libs/node/node.py

```diff
@@ -35,7 +35,10 @@ def wait_for_disk_update(self, node_name, disk_num):
                 all_updated = True
                 disks = node.disks
                 for d in disks:
-                    if disks[d]["diskUUID"] == "":
+                    if disks[d]["diskUUID"] == "" or
+                        not disks[d]["conditions"] or
+                        disks[d]["conditions"]["Ready"]["status"] != "True" or
+                        disks[d]["conditions"]["Schedulable"]["status"] != "True":
```

Comment:
If a block disk in `node.spec.disks`  is not added to the `node.status.diskStatus` yet, the check will succeed. Then. a volume will be created before the disk becomes ready.

Reply:
It assert len(node.disks) == disk_num, and disk_num equals 2 here, which means default disk + newly added block disk.